net/crowdsec/patches/002-fix_unused_plugin.patch

```diff
@@ -0,0 +1,18 @@
+--- a/pkg/parser/enrich.go
++++ b/pkg/parser/enrich.go
```

Comment:
Why this patch is needed? There is missing commit description, commit author, etc.

Reply:
In that case as this is backport from upstream, I suggest taking commit from them and add ``.patch`` in the URL, which leads to this: https://github.com/crowdsecurity/crowdsec/commit/eb80248fa8babd9857e097b13466c23730d53bcf.patch, download it and replace the patch to be more clear about it.

Reply:
Ok !
Done...
Thanks